telco-hub/configuration/reference-crs/required/logging/cLusterLogSubscription.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging
+spec:
+  channel: "stable"
+  name: cluster-logging
+  source: redhat-operators-disconnected
+  sourceNamespace: openshift-marketplace
+  installPlanApproval: Automatic
+status:
+  state: AtLatestKnown
```

Comment:
On the hub we will not be using policy to apply these so we should leave off the status: section. Here and other Subscriptions below.